runtime/service.go

```diff
@@ -484,6 +485,10 @@ func (s *service) publishVMStop() error {
 	return s.eventExchange.Publish(s.shimCtx, StopEventName, &proto.VMStop{VMID: s.vmID})
 }
 
+func jailerRootDir(shimBaseDir, namespace, vmID string) string {
```

Comment:
I wonder if we should move this to `internal/vm` package similarly to all the other directory stuff.

Reply:
I'd like to keep it here. Moving that to internal/vm means that we need to expose the function, which in unnecessary.

Reply:
That's the point of internal. In Go, no one can use that package but the package itself.

Reply:
Yeah. But moving there means anybody in the repo (e.g. agent) can use that. Do we want to?

Reply:
Yea, I mean I don't see why not since it is in internal and it'll be with the rest of the directory stuff, but I don't feel too strongly on it.

Reply:
Oh, there is ShimDir in the internal package which does exactly the same 🤦